app/src/main/java/org/thunderdog/challegram/component/sticker/StickerSetWrap.java

```diff
@@ -377,7 +377,7 @@ public void onFactorChangeFinished (int id, float finalFactor, FactorAnimator ca
 
   private void setInProgress (boolean inProgress) {
     if (this.inProgress != inProgress) {
-      this.inProgress = inProgress;
+      UI.post(() -> this.inProgress = inProgress);
```

Comment:
Outdated. This is no longer needed.
```suggestion
      this.inProgress = inProgress;
```